httpbp/admin.go

```diff
@@ -15,5 +18,9 @@ import (
 // This function blocks, so it should be run as its own goroutine.
 func ServeAdmin(healthCheck HandlerFunc) {
 	admin.Mux.Handle("/health", handler{handle: healthCheck})
-	log.Warnw("httpbp: admin serving exited", "err", admin.Serve())
+	if err := admin.Serve(); errors.Is(err, http.ErrServerClosed) {
+		log.Info("httpbp: server closed")
```

Comment:
actually, can we change this log (and the thriftbp one) to "*admin* server closed"?

the current log sounds like the main httpbp/thriftbp server was closed.

Reply:
done